Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -15,26 +15,8 @@ constexpr double k_ScaleFromDDD = 0.1;
 HGCalDDDConstants::HGCalDDDConstants(const HGCalParameters* hp,
 				     const std::string& name) : hgpar_(hp) {
   mode_ = hgpar_->mode_;
-  if (mode_ == HGCalGeometryMode::Square) {
-    rmax_    = 0;
-    modHalf_ = sectors()*layers(true);
-    for (int simreco = 0; simreco < 2; ++simreco ) {
-      tot_layers_[simreco] = layersInit((bool)simreco);
-    }
-
-    edm::LogInfo("HGCalGeom") << "HGCalDDDConstants initialized for " << name
-			      << " with " << layers(false) << ":" 
-			      << layers(true) << " layers, " << sectors() 
-			      << " sectors " << 2*modHalf_ << " modules and "
-			      << "maximum of " << maxCells(false) << ":" 
-			      << maxCells(true) << " cells";
-#ifdef EDM_ML_DEBUG
-    std::cout << "HGCalDDDConstants initialized for " << name << " with " 
-	      << layers(false) << ":" << layers(true) << " layers, " 
-	      << sectors() << " sectors and maximum of " << maxCells(false)
-	      << ":" << maxCells(true) << " cells" << std::endl;
-#endif
-  } else {
+  if ((mode_ == HGCalGeometryMode::Hexagon) ||
```

Comment:
@bsunanda - there are only these two modes. It's better to use an assert.

Reply:
There will be some more new modes which will come